MaraBot/Core/Preset.cs

```diff
@@ -87,7 +87,7 @@ public void MakeDisplayable(IReadOnlyDictionary<string, Option> options)
                     ));
             }
 
-            Mode mode = Option.OptionValueToMode(Options["mode"]);
+            Mode mode = Options.ContainsKey("mode") ? Option.OptionValueToMode(Options["mode"]) : Mode.Rando;
```

Comment:
There used to be a huge validation function here that checked this lol.
Still have to reimplement that somehow...
Maybe by using a `!verify`?